graphblas/matrix.py

```diff
@@ -136,16 +141,39 @@ def _deserialize(pieces, name):
 
     @property
     def S(self):
+        """
+        Creates a Mask based on the structure of the Matrix.
+        """
         return StructuralMask(self)
 
     @property
     def V(self):
+        """
+        Creates a Mask based on the values in the Matrix (treating each value as truthy).
+        """
         return ValueMask(self)
 
     def __delitem__(self, keys):
+        """
+        Delete a single element, row/column, or submatrix.
+
+        .. code-block:: python
+
+            del M[1, 5]
+        """
         del Updater(self)[keys]
 
     def __getitem__(self, keys):
+        """
+        Extract a single element, row/column, or submatrix.
+        The `user guide <../user_guide/operations.html#extract>`__ contains more details.
```

Comment:
We really ought to try to follow PEP 257 and have a one liner at the beginning followed by a newline:

https://peps.python.org/pep-0257/

Reply:
Interesting that [numpy prefers PEP 257 style](https://numpydoc.readthedocs.io/en/latest/format.html) while [pandas prefers the style I have been writing](https://pandas.pydata.org/docs/development/contributing_docstring.html).

I'm fine switching to the numpy style.

Reply:
Pandas style is great, and it still prefers a single line short summary at the beginning followed by empty line:
https://pandas.pydata.org/docs/development/contributing_docstring.html#section-1-short-summary